packages/tools/kadena-cli/src/devnet/commands/devnetSimulation.ts

```diff
@@ -0,0 +1,52 @@
+import type { ChainId } from '@kadena/types';
+import chalk from 'chalk';
+import debug from 'debug';
+import type { CreateCommandReturnType } from '../../utils/createCommand.js';
+import { createCommand } from '../../utils/createCommand.js';
+import { globalOptions } from '../../utils/globalOptions.js';
+import { containerIsRunning, guardDocker } from '../utils/docker.js';
+import { simulateCoin } from '../utils/simulation/coin/simulate.js';
+
+export const simulateCommand: CreateCommandReturnType = createCommand(
+  'simulate',
+  'Simulate traffic on a devnet',
+  [
+    globalOptions.devnet(),
+    globalOptions.numberOfAccounts({ isOptional: true }),
+    globalOptions.transferInterval({ isOptional: true }),
+    globalOptions.logFolder({ isOptional: true }),
+    globalOptions.tokenPool({ isOptional: true }),
+    globalOptions.maxTransferAmount({ isOptional: true }),
+    globalOptions.defaultChainId({ isOptional: true }),
+    globalOptions.seed({ isOptional: true }),
+  ],
+  async (config) => {
+    try {
+      debug('devnet-simulate:action')({ config });
+
+      guardDocker();
+
+      if (!containerIsRunning(config.devnetConfig.name)) {
+        console.log('Devnet is not running. Please run the devnet first.');
+        return;
+      }
+
+      await simulateCoin({
+        network: {
+          id: 'fast-development',
+          host: `http://localhost:${config.devnetConfig.port}`,
+        },
```

Comment:
can we use `globalOptions.network()` here?

Reply:
We can indeed use it. However, since I already prompt the user to choose a `devnet` I don't know if it's necessary to have both of the options. The reason for the `devnet` prompt instead of `network`, is that we can validate if the selected devnet configuration is running (by verifying the running containers against the devnet name)

Reply:
Simulation should be network independent. So it can be any network that's running anywhere, not just on `localhost`.